kolibri_zim_plugin/assets/src/views/ZimContentView.vue

```diff
@@ -191,13 +170,15 @@
     width: 100%;
     height: 100%;
     overflow: auto;
+    opacity: 1;
+    transition: opacity 0.5s ease;
```

Comment:
This transition should be removed, it shows the unstyled content to new content transition as a fade-out & fade-in, without the transtion works correctly for me in chromium, but with this line I can see the unstyled content before the correct presentation.